packit/base_git.py

```diff
@@ -415,6 +419,18 @@ def set_specfile_content(
             self.specfile.version = version
         if comment is not None:
             self.specfile.add_changelog_entry(comment)
+        if (
+            previous_version != self.specfile.expanded_version
+            and previous_release == self.specfile.release
+        ):
+            # This may occur if the upstream forgets to reset release after
+            # bumping the version, or if the specfile is not maintained in
+            # upstream at all (e.g. post-upstream-clone that uses wget to
+            # get the specfile from Fedora).
+            # Either way, it seems better to fix it at least in downstream
+            # at the cost of upstream and downstream having different
+            # specfiles.
```

Comment:
in this case, different "Release" fields, that are only relevant within one buildsystem :)

+1